tools/expotools/src/commands/UpdateVendoredModule.ts

```diff
@@ -503,6 +503,29 @@ async function askForModuleAsync(): Promise<string> {
   return moduleName;
 }
 
+async function getPackageJsonPathsAsync(): Promise<string[]> {
+  const packageJsonPath = path.join(Directories.getAppsDir(), '**', 'package.json');
```

Comment:
What about home? 😄 Also, if anything has changed in home's dependencies I think we should log that publishing new dev home might be required after testing.